include/mbgl/ios/MGLConfigurationManager.h

```diff
@@ -0,0 +1,43 @@
+#import "MGLTypes.h"
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** The MGLConfigurationManager object provides a global way to set a Mapbox API access token, as well as other settings used framework-wide. */
+@interface MGLConfigurationManager : NSObject
+
+/** @name Authorizing Access */
+
+/** Set the Mapbox API access token for the framework.
+*
+*   You can set an access token on MGLConfigurationManager or on an individual map view. The same token is used throughout the framework.
+*   @param accessToken The Mapbox API access token. */
++ (void)setAccessToken:(nullable NSString *)accessToken;
+
+/** Retreive the Mapbox API access token for the framework.
+*
+*   You can set an access token on MGLConfigurationManager or on an individual map view. The same token is used throughout the framework.
+*   @return accessToken The Mapbox API access token. */
++ (nullable NSString *)accessToken;
+
+/** @name Providing User Metrics Opt-Out */
++ (void)setMapboxMetricsEnabledSettingShownInApp:(BOOL)showsOptOut __attribute__((unavailable("Set MGLMapboxMetricsEnabledSettingShownInApp in Info.plist.")));
```

Comment:
Since MGLConfigurationManager is replacing MGLAccountManager, this is a good time to remove obsoleted methods. (We marked them `unavailable` instead of `deprecated` in light of being pre-1.0, when backwards compatibility is less of a concern.)